patch-package
--- a/node_modules/react/cjs/react.development.js
+++ b/node_modules/react/cjs/react.development.js
@@ -58,7 +58,7 @@ function getIteratorFn(maybeIterable) {
 // render phase
 
 // Suspense
-var enableSuspense = false;
+var enableSuspense = true;
 // Helps identify side effects in begin-phase lifecycle hooks and setState reducers:
 
 
--- a/node_modules/react/umd/react.development.js
+++ b/node_modules/react/umd/react.development.js
@@ -55,7 +55,7 @@ function getIteratorFn(maybeIterable) {
 // render phase
 
 // Suspense
-var enableSuspense = false;
+var enableSuspense = true;
 // Helps identify side effects in begin-phase lifecycle hooks and setState reducers:
 
 
